packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorBy,
+    WaferMapColorsScale,
+    WaferMapDataType,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'wafer-quadrant'
+    })
+    public waferQuadrant!: WaferMapQuadrant;
```

Comment:
Could you provide a default value for all the simple types? That way you can remove the `!` since the value can never be null/undefined.

Reply:
Added